Amplitude/Amplitude.m

```diff
@@ -31,30 +31,42 @@
 @interface Amplitude ()
 
 @property (nonatomic, retain) NSOperationQueue *backgroundQueue;
+@property (nonatomic, retain) NSOperationQueue *initializerQueue;
 @property (nonatomic, retain) NSMutableDictionary *eventsData;
 @property (nonatomic, assign) BOOL initialized;
 @property (nonatomic, assign) BOOL sslPinningEnabled;
+@property (nonatomic, assign) long long sessionId;
 
 @end
 
+NSString *const kAMPSessionStartEvent = @"session_start";
+NSString *const kAMPSessionEndEvent = @"session_end";
+NSString *const kAMPRevenueEvent = @"revenue_amount";
+
 @implementation Amplitude {
-    AMPDeviceInfo *_deviceInfo;
-    NSString *_eventsDataPath;
-    NSOperationQueue *_initializerQueue;
+
     NSOperationQueue *_mainQueue;
+
+    AMPDeviceInfo *_deviceInfo;
+
     CLLocation *_lastKnownLocation;
     BOOL _locationListeningEnabled;
     CLLocationManager *_locationManager;
     AMPLocationManagerDelegate *_locationManagerDelegate;
+
+    NSString *_eventsDataPath;
     NSMutableDictionary *_propertyList;
     NSString *_propertyListPath;
-    long long _sessionId;
-    BOOL _sessionStarted;
+
     BOOL _updateScheduled;
     BOOL _updatingCurrently;
     UIBackgroundTaskIdentifier _uploadTaskID;
+
```

Comment:
It separates out variables that are related to each other. I reorganized them so there are fewer spaces.